contrib/osx/make_osx.sh

```diff
@@ -256,4 +255,4 @@ DoCodeSignMaybe ".DMG" "dist/electrum-${VERSION}.dmg"
 if [ -z "$CODESIGN_CERT" ]; then
     warn "App was built successfully but was not code signed. Users may get security warnings from macOS."
     warn "Specify a valid code signing identity to enable code signing."
-fi
+fi
```

Comment:
please fix the code formatting issues so that the flake8 tests pass
https://cirrus-ci.com/task/4585331354238976

for example, "no newline at end of file"